api/src/main/java/org/openmrs/module/exportccd/api/generators/MedicationSectionGenerator.java

```diff
@@ -47,29 +52,53 @@ public ContinuityOfCareDocument buildMedication(ContinuityOfCareDocument ccd, Pa
 	}
 	
 	private String generateDrugSectionContent(Patient patient) {
-		StringBuilder builder = utils.buildSectionHeader("Medication", "Date", "Dose", "Days");
+		StringBuilder builder = utils.buildSecondHeaderRow("Medication/Posologie", "Début", "Fin",
```

Comment:
It can be 'buildSectionHeader' i think. then method 'buildSecondHeaderRow' could be deleted, due to no ussages